backend/de.metas.material/cockpit/src/main/java/de/metas/material/cockpit/availableforsales/AvailableForSalesSqlHelper.java

```diff
@@ -71,7 +70,7 @@
 		{
 			return queryBuilder
 					.addOnlyActiveRecordsFilter()
-					.addEqualsFilter(I_MD_Available_For_Sales_QueryResult.COLUMN_M_Product_ID, availableForSalesQuery.getProductId())
+					.addEqualsFilter(I_MD_Available_For_Sales_QueryResult.COLUMN_M_Product_ID, availableForSalesQuery.getProductId().getRepoId())
```

Comment:
```suggestion
					.addEqualsFilter(I_MD_Available_For_Sales_QueryResult.COLUMN_M_Product_ID, availableForSalesQuery.getProductId())
```

equalsFilter can work with Repoawares like ProdcutId directly.

Reply:
Done.